### ISSUE TITLE
How to get tracked surfaces

### ISSUE THREAD

**Issue by carlesgutierrez** [CariusLars/ar_flutter_plugin#208]
Hello everyone,
  My goal is to analyze tracked surfaces from the device and then be able to place as many objects as I can automatically.
 
   These types of methods (get trackable surfaces) are already available on other platforms as a Unity [TrackableType class.](https://docs.unity3d.com/Packages/com.unity.xr.arfoundation@5.0/api/UnityEngine.XR.ARSubsystems.TrackableType.html)  or also as a [DebugDrawTrackedGeometry](https://docs.unrealengine.com/4.26/en-US/BlueprintAPI/ARAugmentedReality/Debug/DebugDrawTrackedGeometry/)  UE5 class.

I suppose it is not a method that is commonly used or I have not been able to find it here. Are these surfaces available at [ar_flutter_plugin](https://github.com/CariusLars/ar_flutter_plugin)?
 
Thanks for reading,
Carles

**Comment by jaysephjw** [comment]
Are these available in ARKit and/or ARCore?

**Comment by antont** [comment]
> Are these available in ARKit and/or ARCore?

Yes, that's the doc for plane detection and tracking in ARKit, https://developer.apple.com/documentation/arkit/arkit_in_ios/content_anchors/tracking_and_visualizing_planes
